Gemfile

```diff
@@ -1,6 +1,12 @@
 # frozen_string_literal: true
 source "https://rubygems.org"
 
+gem 'actionmailer', '~> 5.2.0'
+gem 'actionpack', '~> 5.2.0'
+gem 'activemodel', '~> 5.2.0'
+gem 'activerecord', '~> 5.2.0'
+gem 'activesupport', '~> 5.2.0'
+
```

Comment:
Don't these automatically come with rails?

Reply:
They do, but upgrading dependencies via `bundler` wasn't elegant.

I'll remove these lines, and I think we should be fine as long as these changes are remarked in `Gemfile.lock`.